src/test/java/mpack/Assignement1.java

```diff
@@ -0,0 +1,58 @@
+//Invoke firefox browser
+package mpack;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.interactions.Actions;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+public class Assignement1 {
+
+	public static void main(String[] args) throws InterruptedException {
+		// Setup chromedriver
+		WebDriverManager.firefoxdriver().setup();
```

Comment:
If possible move this to base class and call it from there instead of writing it in each class .This will reduce common /duplicate code

Reply:
As suggested shifted common code in Base class